test/utils/index.js

```diff
@@ -54,7 +54,8 @@ function exec (args, options, callback) {
   options = options || {};
   var env = _.extend({}, options.env, process.env);
 
-  return child_process.exec("node " + jpm + " " + args, {
+  return child_process.exec("node " + jpm + " --addon-dir " + options.addonDir
```

Comment:
what happens if `options.addonDir` is undefined? That could potentially break a test no? I think it would be a little safer as:
````
if (options.addonDir) {
  args = "--addon-dir " + options.addonDir + " " + args;
}
return child_process.exec("node " + jpm + " " + args ...
````